real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -327,5 +357,27 @@ public static double getPlayBackSpeed() {
     public static void setPlayBackSpeed(double playBackSpeed) {
         RobobuggyConfigFile.playBackSpeed = playBackSpeed;
     }
+    
+    /**
+     * @return the robottype (whatever's in the config file)
+     */
+    public static String getRobotType() {
+        return robotType;
+    }
+
+    /**
+     * @param robottype the robottype to set
+     */
+    public static void setRobotType(String robottype) {
```

Comment:
I'm not too keen on the idea of being able to dynamically set the robot type. This could cause issues with how the rest of the system behaves